cli/crud/import/import.go

```diff
@@ -0,0 +1,1214 @@
+/*
+	The batches are transmitted to the router in the form of a serialized json (struct t_batch).
+	Feedback from crud is obtained through this way also.
+
+	At the end of the program batches amount will be ⌈csvRecodrdsAmount / batchSize⌉, where csvRecodrdsAmount
+	is the number of csv entries (both correct and incorrect syntax) in the input data.
+	Example: 10 correct entries (8 of them will be imported, and 2 will be not due to some crud error),
+	and 3 with a syntax error. csvRecodrdsAmount: 8 + 2 + 3 = 13; batchSize: 5; batches amount will be ⌈13 / 5⌉ = 3.
+	Finally, batches amount = number of batch uploading to the router = number of insert_many/replace_many iterations.
+*/
+
+package crudimport
+
+import (
+	"bufio"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/signal"
+	"strconv"
+	"syscall"
+	"time"
+
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/connector"
+	ttcsv "github.com/tarantool/tt/cli/ttparsers"
+	"github.com/tarantool/tt/cli/util"
+	"gopkg.in/yaml.v2"
+)
+
+// t_importSummary contains information for the import summary.
```

Comment:
What does `t_` mean? In golang we (and not only) do not use snake_case.
Here and below.

Reply:
Postfix as an abbreviation of the word `type`
Now using postfix `Container` for naming structs; defined all related types in `cli/crud/import/containers.go`